_content/lorawan/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: LoRaWAN
+description: Background information about LoRaWAN
+image: /devices/icon.png
+zindex: 1100
+---
+
+# LoRaWAN Overview
```

Comment:
Should we add the What's new in LoRaWAN 1.1 video link here somewhere, or in the bottom?